.circleci/local_publish_helpers.sh

```diff
@@ -28,9 +58,14 @@ function generatePkgCli {
   # Transpile code for packaging
   npx babel node_modules --extensions '.js,.jsx,.es6,.es,.ts' --copy-files --include-dotfiles -d ../build/node_modules
 
+  # Include third party licenses
+  cp ../Third_Party_Licenses.txt ../build/node_modules
```

Comment:
Third_Party_Licenses are bundled into the resulting binary. Currently keeping this file up to date is a manual process, but the AWS CDK team is building a tool called `node_bundle` that has the ability to generate this attributions file automatically. When it is ready, our manual process can be replaced with a pre commit hook that updates these attributions as depedencies change.